docs/application/tizen-studio/release-notes/4-1-release-notes.md

```diff
@@ -23,6 +23,7 @@
 - Tizen Studio
   - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
   - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
```

Comment:
@manishtr Only for native project : `tizen build -c gcc` has an issue with old CLI ?